### PR TITLE
ISSUE TEMPLATE ADDED : fixes #59

### LINKED ISSUE #59: No Issues template
The issues are unstructured. 

They need a template.

### PR DESCRIPTION
**Pull request by YashKumarVerma**
This pull request is related to issue #59  

I modified the labyrinth, you can test my changes here:
http://rawgit.com/yashkumarverma/labyrinth/master/index.html

The problem I want to solve is **that there is no issues template**

My solution for the problem is **i created one**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Expected results
+
+### Actual results
+
+### Steps to reproduce the issue
```
Comment (abishekvashok) [inline .github/ISSUE_TEMPLATE.md]: Can you add comments here?

Comment (YashKumarVerma) [reply .github/ISSUE_TEMPLATE.md]: More comments as in?
Please suggest

**Comment by niccokunzmann** [comment]
Cool, thanks ... anyone who like to improve it can now create an other pull request.

**Comment by YashKumarVerma** [comment]
@niccokunzmann i'd be adding more comments if required / or if suggested.

Thanks 😃

**Comment by niccokunzmann** [comment]
sure :)

**Pull request merged**